src/debug/di/rsmain.cpp

```diff
@@ -74,7 +74,12 @@ const char * GetDebugCodeName(DWORD dwCode)
 // Per-thread state for Debug builds...
 //-----------------------------------------------------------------------------
 #ifdef RSCONTRACTS
-DWORD DbgRSThread::s_TlsSlot = TLS_OUT_OF_INDEXES;
+#ifndef __GNUC__
```

Comment:
This is following the existing pattern. It would be nice to have a define for this instead of these `__GNUC__` ifdefs, but finding the right places to put it needs some experimentation. Leaving it for separate change.